src/TouristInfoCards.js

```diff
@@ -0,0 +1,59 @@
+          alt=""
+          className="card-img-top"
+        />
+        <div className="card-body">
+          <a
+            href="https://peoplemakeglasgow.com/"
```

Comment:
It would be better if the information here is passed in through using a `prop` rather than put directly into the file.

Could you extra the URL, Image and Button Text from here into a JavaScript object file similar to `FakeBookings.js`? Then you can import it in the `Bookings.js` file and pass it as a `prop`.

Reply:
Correction done
Thanks Chris